### PR TITLE
refac: refactor prefixes and adding namespaces

### PR DESCRIPTION
**Pull request by andpou**
# Description

## Pull-request quality

- [ ] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

**Review by x-platformcoder — DISMISSED** [review]
LGTM

File: source/measurements/contracts/PersistSubmittedTransaction.proto [hunk 1]
```diff
@@ -4,18 +4,21 @@ package google.protobuf;
...
   MeteringPointType metering_point_type = 6;
-  Unit unit = 7;
-  Resolution resolution = 8;
-  Timestamp start = 9;
-  Timestamp end = 10;
-  repeated Point points = 11;
+  string product = 7;
+  Unit unit = 8;
+  Resolution resolution = 9;
+  Timestamp start_datetime = 10;
```
Comment (x-platformcoder) [inline source/measurements/contracts/PersistSubmittedTransaction.proto:19]: Mandalorian har forespurgt om vi kan flytte start_datetime og end_datetime op så de er efter transaction_creation_datetime dvs. pos. 6+7.

Comment (awha86) [reply source/measurements/contracts/PersistSubmittedTransaction.proto]: done

**Review by x-platformcoder — APPROVED** [review]
lgtm

**Pull request merged**